apis/addons/v1alpha1/addons_types.go

```diff
@@ -46,6 +46,19 @@ type AddonSpec struct {
 
 	// Settings for propagating secrets from the Addon Operator install namespace into Addon namespaces.
 	SecretPropagation *AddonSecretPropagation `json:"secretPropagation,omitempty"`
+
+	// defines the PackageOperator image as part of the addon Spec
+	AddonPackageOperator *AddonPackageOperator `json:"addonPackageOperator,omitempty"`
+}
+
+type AddonsPackageOperatorImage struct {
+	Name string `json:"name"`
+
+	Image string `json:"image"`
+}
+
+type AddonPackageOperator struct {
```

Comment:
Is this needed as I see you already specified `type AddonsPackageOperatorImage struct` ? :)

Reply:
@yashvardhan-kukreja  really I am trying to achieve this ```kind: Addon
spec:
  install:
    packageOperator:
      image: xxx```

Reply:
Ahh, then, any reason why you are creating the `type AddonsPackageOperatorImage struct` which has both `name and image` fields